### PR TITLE
chore(changelog): Update changelog for preview5

### PR DESCRIPTION
**Pull request by ixje**

### CODE REVIEW

File: docs/changelog/latest.md [hunk 1]
```diff
@@ -1,9 +1,33 @@
 ---
 id: latest
-title: Changelog (v5)
+title: Changelog (v6)
```
Comment (snowypowers) [inline docs/changelog/latest.md:3]: still v5.

Comment (ixje) [reply docs/changelog/latest.md:3]: oh I thought the version would go to `next.6`.

File: docs/changelog/latest.md [hunk 2]
```diff
@@ -3,7 +3,31 @@ id: latest
 title: Changelog (v5)
 ---
 
-# 5.0.0
+# 6.0.0
```
Comment (snowypowers) [inline docs/changelog/latest.md:6]:
```suggestion
# 5.0.0
```

File: docs/changelog/latest.md (hunk 1 above)
Comment (snowypowers) [reply docs/changelog/latest.md]: next6 !== v6. The version here tracks the major version. The expected official production version for neo3 is v5.0.0 .

**Review by snowypowers — APPROVED** [review]

**Pull request merged**